GameData/RP-1/Strategies/Leaders/LeadersSubcontractors.cfg

```diff
@@ -25,41 +25,17 @@ STRATEGY
 	{
 		research_tech = postWarMaterialsScience
 	}
-
+	
 	EFFECT
 	{
-		name = CurrencyModifier
-		effectTitle = No Need to Innovate
-		effectDescription = researcher salaries
-		currency = Funds
-		multiplier = 0.95
-		flipPositive = true
-		transactionReasons {
-			item = SalaryResearchers
-		}
-	}
-	EFFECT
-	{
-		name = CurrencyModifier
-		effectTitle = Shared Prestige
-		currency = Reputation
-		effectDescription = from contracts
-		multiplier = 0.95
-		transactionReasons
-		{
-			item = ContractReward
-		}
-	}
-	EFFECT
-	{
-		name = CurrencyModifier
-		effectTitle = ОКБ Politics
-		currency = Confidence
-		effectDescription = from contracts
-		multiplier = 0.95
-		transactionReasons
+		name = IntegrationRateModifier
+		effectTitle = Mass Production (NEEDS TESTING)
+		effectDescription = of conventional structure tanks
+		multiplier = 4.0
+		appliesToParts = true
+		tags
 		{
-			item = ContractReward
+			item = TankSeparate 
```

Comment:
There is no such tag. Tags are defined here: https://github.com/KSP-RO/RP-1/blob/master/GameData/RP-1/SCMData/CostModifiers.cfg